tasks/travis-pr-title-lint.js

```diff
@@ -18,25 +18,53 @@ var writeCommitMessagesDocURL =
   '/CONTRIBUTING.md#writing-commit-messages';
 
 module.exports = function(grunt) {
-  function countGitMergeCommits() {
+  function findMergeBase() {
     return new Promise(function(resolve, reject) {
-      exec('git rev-list --count HEAD ^master', function(err, stdout) {
+      exec('git merge-base HEAD master', function(err, stdout) {
         if (err) {
           reject(err);
         } else {
-          resolve(parseInt(stdout));
+          var baseCommit = stdout.trim();
+          if (process.env.VERBOSE === 'true') {
+            console.log('DEBUG findMergeBase:', baseCommit); // eslint-disable-line no-console
+          }
+          resolve(baseCommit);
         }
       });
     });
   }
 
-  function getGitLastCommitMessage() {
+  function getGitBranchCommits() {
     return new Promise(function(resolve, reject) {
-      exec('git show -s --format=%B HEAD', function(err, stdout) {
+      findMergeBase().then(function(baseCommit) {
+        var gitCommand = 'git rev-list --no-merges HEAD ^' + baseCommit;
+        exec(gitCommand, function(err, stdout) {
+          if (err) {
+            reject(err);
+          } else {
+            var commits = stdout.trim().split('\n');
+            if (process.env.VERBOSE === 'true') {
+              console.log('DEBUG getGitBranchCommits:', commits); // eslint-disable-line no-console
+            }
+            resolve(commits);
```

Comment:
Instead of resolving the "number of commit", it returns the array of the commits (so that we can be able to retrieve the commit message from the pull request, because HEAD is actually the merge commit and HEAD^ is the last commit on master)